whynotearth/Volkswagen-email-corporate-communications-CMS
# 411 Stats overview distribution group

# Issue Being Addressed

#411 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

- [x] Update to Existing Feature
- [ ] Other (state below)

# Description

_For Bug Fixes:_ Fix percent colors in charts (red and green)

_For New Features:_ Add stats overview for distribution groups

_For Feature Updates:_ Update link of items in distribution groups page to point to stats overview

# How to Test/Reproduce

1. Go to settings and distribution groups
2. Click on every group and see the stats page

# Screenshots/casts

![localhost_8080_stats_distribution-groups_Dev%20Test](https://user-images.githubusercontent.com/510242/84805620-c8566e80-b019-11ea-81fb-59b98ec95363.png)


# Additional Context

Nothing is connected to api on overview stats page yet.  
Edit List Name will be done in another PR https://github.com/whynotearth/Volkswagen-email-corporate-communications-CMS/issues/366
